### ISSUE TITLE
[Bug]: SSH does not work anymore in version 2.9.0

### ISSUE THREAD

**Issue by davidkopp** [denolehov/obsidian-git#361]
### Describe the bug

After upgrading from 2.8.0 to 2.9.0 the SSH connection does not work anymore. When I go back to 2.8.0 it works again.
I checked the code changes but I have no idea what could cause the issue.

### Relevant errors (if available)

```shell
git obsidian error: git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

### Steps to reproduce

Execute an action that connects to the remote repository via SSH (e.g. `Obsidian Git: Pull`)

### Operating system

Windows

### Plugin version

2.9.0

**Comment by Vinzent03** [comment]
Should be fixed now. Released 2.9.1

**Comment by davidkopp** [comment]
yes, it works again :)
